client/packages/system/src/utils.ts

```diff
@@ -42,3 +43,37 @@ export const masterListsToCsv = (
   ]);
   return Formatter.csv({ fields, data });
 };
+
+export const stockLinesToCsv = (
+  invoices: StockLineRowFragment[],
+  t: TypedTFunction<LocaleKey>
+) => {
+  const fields: string[] = [
+    'id',
+    t('label.code'),
+    t('label.name'),
+    t('label.batch'),
+    t('label.expiry'),
+    t('label.location'),
+    t('label.unit'),
+    t('label.pack-size'),
+    t('label.num-packs'),
+    t('label.available-packs'),
+    t('label.supplier'),
+  ];
+
+  const data = invoices.map(node => [
+    node.id,
+    node.item.code,
+    node.item.name,
+    node.batch,
+    Formatter.csvDateString(node.expiryDate),
```

Comment:
This includes a time in the output date string, e.g. `2024-07-27 12:00:00`. Is this what we want? (Wouldn't just the date be more appropriate?)

Reply:
I'll add an expiryDateString helper..